brands/shop/beauty.json

```diff
@@ -174,5 +174,15 @@
       "name": "Yves Rocher",
       "shop": "beauty"
     }
+  },
+  "shop/beauty| Carlance": {
+    "countryCodes": ["fr"],
+    "tags": {
+      "beauty" : "hair;nails;skin_care;massage",
+      "brand" : "Carlance",
+      "brand:wikidata" : "Q91601625",
+      "name" : "Carlence",
```

Comment:
This is Carlance, not Carlence

Reply:
I think I have successfully done the changes..please let me know

Thanks